src/plugins/guided_onboarding/public/components/guide_panel.styles.ts

```diff
@@ -23,18 +23,19 @@ export const getGuidePanelStyles = (euiTheme: EuiThemeComputed) => ({
   flyoutOverrides: {
     flyoutContainer: css`
       top: 55px !important;
-      bottom: 25px !important;
+      bottom: unset !important;
```

Comment:
nit: would it be possible to delete the property completely?

Reply:
(If it's not possible to delete, it might be worth adding a comment to explain why.)

Reply:
@yuliacech @alisonelizabeth - good point, I added a comment to clarify (does not work if we remove these values) - I had one quick question, is this placement the correct place to put it, or should I add it all the way at the top with the commented description:
```
/**
 *
 * Style overrides for the setup guide dropdown panel.
 * There is currently no existing EUI component that fully supports what we need.
 ```

Reply:
I think this is OK 👍. Not sure if there are any specific guidelines around this, but in this case, I think it's easier to follow next to the CSS.